packages/core/core-flows/src/product/workflows/update-products.ts

```diff
@@ -248,9 +247,14 @@ export const updateProductsWorkflow = createWorkflow(
     })
 
     dismissRemoteLinkStep(toDeleteSalesChannelLinks)
-
     createRemoteLinkStep(salesChannelLinks)
 
-    return new WorkflowResponse(updatedProducts)
+    const productsUpdated = createHook("productsUpdated", {
+      products: updatedProducts,
```

Comment:
question: Should this have additional data too?

For example, you might want to update the associated `brand_id` of the product.

Reply:
Yes. It should. Maybe I missed it

Reply:
Fixed